### PR TITLE
Make plugin work with Vanilla 2.6

### PR DESCRIPTION
**Pull request by R-J**
- Fix class name
- Replace `AddSideMenu` with `SetHighlightRoute`
- Replace Plugin->GetView with Controller->Render
- Bump plugin and required Vanilla version

### CODE REVIEW

File: src/main/php/class.nillablog.plugin.php [hunk 1]
```diff
@@ -16,22 +16,22 @@
  */
 
 $PluginInfo["NillaBlog"] = array(
-	"Name" => "${project.plugin.name}",
-	"Description" => "${project.description}",
-	"Version" => "${project.version}",
-	"Author" => "${project.author.name}",
-	"AuthorEmail" => "${project.author.email}",
-	"AuthorUrl" => "${project.url}",
+	"Name" => "NillaBlog",
```
Comment (ddumont) [inline src/main/php/class.nillablog.plugin.php]: Can you please revert lines 19-24? The build I do will take care of that.

**Comment by R-J** [comment]
Done!

**Comment by ddumont** [comment]
whoa, lots of changes here.   Can you verify this works as-is?   I was just expecting you to revert the plugin header

**Comment by R-J** [comment]
Oops, sorry! I've totally forgotten that I had started to change it to current coding standards - for no real reason. As an excuse: between my first and my last commit has been a surgery and looots of drugs.
I close this PR and create a new one. My limited Git knowledge makes this the fastest solution

**Comment by ddumont** [comment]
No worries. I would have merged it, but I wanted an additional confirmation
that everything still worked.

**Pull request closed without merging**